packages/cloudflare/src/cli/build/patches/to-investigate/patch-find-dir.ts

```diff
@@ -1,25 +1,26 @@
 import { existsSync } from "node:fs";
 import { join } from "node:path";
 
-import { Config } from "../../../config.js";
+import { type BuildOptions, getPackagePath } from "@opennextjs/aws/build/helper.js";
 
 /**
- * Here we patch `findDir` so that the next server can detect whether the `app` or `pages` directory exists
- * (source: https://github.com/vercel/next.js/blob/ba995993/packages/next/src/lib/find-pages-dir.ts#L4-L13)
- * (usage source: https://github.com/vercel/next.js/blob/ba995993/packages/next/src/server/next-server.ts#L450-L451)
- * Note: `findDir` uses `fs.existsSync` under the hood, so patching that should be enough to make this work
+ * Patch `findDir` so that the next server can detect whether the `app` or `pages` directory exists
```

Comment:
```suggestion
 * Patches `findDir` so that the next server can detect whether the `app` or `pages` directory exists
```

but I think we might have different conventions for jsdocs, I am not a fan of these comments being imperative, anyways if you strongly prefer imperative comments that's fine, but at the very least we should be consistent thought the codebase (I can see that we're not 🥲)